src/execute.c

```diff
@@ -19,6 +19,9 @@
 #ifdef HAVE_CONFIG_H
 #include "../config.h"
 #endif /* HAVE_CONFIG_H */
+#ifdef HAVE_STDATOMIC_H
```

Comment:
Instead of adding an include of `<stdatomic.h>`, perhaps we should be removing the include of `<atomic.h>`?  Neither appear to be used in this module.

Reply:
oh, now you want to remove atomic.h ;) 
I am all for it, but given the tone of add not replace I figured it was best left alone.

Reply:
It seems best to remove it if it is not being used.